### PR TITLE
feat: 🎸 Add method to get owner of a NFT

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

* Updates the polkadot dependencies with chain 6.1.1
* Adds a new method `getOwner` to Nft to get the holder of the NFT

### Breaking Changes

NA

### JIRA Link

DA-997

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

**Review by VictorVicente — APPROVED** [review]

File: src/api/entities/Asset/NonFungible/Nft.ts [hunk 1]
```diff
@@ -198,6 +208,71 @@ export class Nft extends Entity<NftUniqueIdentifiers, HumanReadable> {
     return null;
   }
 
+  /**
+   * Get owner of the NFT
+   *
+   * @note This method returns `null` if there is no existing holder for the token. This may happen even if the token has been redeemed/burned
+   */
+  public async getOwner(): Promise<DefaultPortfolio | NumberedPortfolio | null> {
```
Comment (polymath-eric) [inline src/api/entities/Asset/NonFungible/Nft.ts:216]: nft.exists should use this method and return false for redeemed NFTs

File: src/api/entities/Asset/NonFungible/Nft.ts [hunk 2]
```diff
@@ -198,6 +208,71 @@ export class Nft extends Entity<NftUniqueIdentifiers, HumanReadable> {
...
+      id,
+      context: {
+        polymeshApi: {
+          query: { portfolio },
+        },
+      },
+      context,
+    } = this;
+
+    const owner = await this.getOwner();
```
Comment (polymath-eric) [inline src/api/entities/Asset/NonFungible/Nft.ts:259]: I think this check should be encapsulated in the `exists` method, and here just check that method

**Review by polymath-eric — CHANGES_REQUESTED** [review]
should use `owner` method in `exists` check, then in `isLocked` we can use `exists` for the DataUnavailable error check

File: src/api/entities/Asset/NonFungible/Nft.ts (hunk 2 above)
Comment (prashantasdeveloper) [reply src/api/entities/Asset/NonFungible/Nft.ts:244]: We need the owner value to check locked status

**Review by polymath-eric — APPROVED** [review]

**Pull request merged**